rust/metaplex/program/src/instruction.rs

```diff
@@ -109,19 +109,20 @@ pub enum MetaplexInstruction {
     ///   5. `[writable]` Vault account
     ///   6. `[writable]` Fraction mint of the vault
     ///   7. `[]` Auction
-    ///   8. `[]` Your BidderMetadata account
-    ///   9. `[signer optional]` Your Bidder account - Only needs to be signer if payer does not own
-    ///   10. `[signer]` Payer
-    ///   11. `[]` Token program
-    ///   12. `[]` Token Vault program
-    ///   13. `[]` Token metadata program
-    ///   14. `[]` Store
-    ///   15. `[]` System
-    ///   16. `[]` Rent sysvar
-    ///   17. `[]` PDA-based Transfer authority to move the tokens from the store to the destination seed ['vault', program_id, vault key]
+    ///   8. `[]` Auction extended
```

Comment:
Yah again, can't do this - you'll break older UIs. Got to put it at the end, or make a new end point.